src/kata4.humanCatDogYears.js

```diff
@@ -1,5 +1,28 @@
 const humanCatDogYears = (number) => {
```

Comment:
Good job on this one. Ideally, we should try to make sure our code is well structured (indentation, spacing and variables well named).